js/huobi.js

```diff
@@ -6685,11 +6796,55 @@ module.exports = class huobi extends Exchange {
         //        value: '194059884.1850000000000000'
         //    }
         //
-        const timestamp = this.safeNumber (interest, 'ts');
+        // fetchOpenInterest: USDT-M
+        //
+        //     {
+        //         "volume": 7192610.000000000000000000,
+        //         "amount": 7192.610000000000000000,
+        //         "symbol": "BTC",
+        //         "value": 134654290.332000000000000000,
+        //         "contract_code": "BTC-USDT",
+        //         "trade_amount": 70692.804,
+        //         "trade_volume": 70692804,
+        //         "trade_turnover": 1379302592.9518,
+        //         "business_type": "swap",
+        //         "pair": "BTC-USDT",
+        //         "contract_type": "swap",
+        //         "trade_partition": "USDT"
+        //     }
+        //
+        // fetchOpenInterest: COIN-M Swap
+        //
+        //     {
+        //         "volume": 518018.000000000000000000,
+        //         "amount": 2769.675777407074725180,
+        //         "symbol": "BTC",
+        //         "contract_code": "BTC-USD",
+        //         "trade_amount": 9544.4032080046491323463688602729806842458,
+        //         "trade_volume": 1848448,
+        //         "trade_turnover": 184844800.000000000000000000
+        //     }
+        //
+        // fetchOpenInterest: COIN-M Futures
+        //
+        //     {
+        //         "volume": 118850.000000000000000000,
+        //         "amount": 635.502025211544374189,
+        //         "symbol": "BTC",
+        //         "contract_type": "this_week",
+        //         "contract_code": "BTC220930",
+        //         "trade_amount": 1470.9400749347598691119206024033947897351,
+        //         "trade_volume": 286286,
+        //         "trade_turnover": 28628600.000000000000000000
+        //     }
+        //
+        const timestamp = this.safeInteger (interest, 'ts');
         return {
             'symbol': this.safeString (market, 'symbol'),
-            'baseVolume': this.safeNumber (interest, 'volume'),
-            'quoteVolume': this.safeValue (interest, 'value'),
+            'baseVolume': this.safeNumber (interest, 'volume'),  // deprecated
```

Comment:
@Dan-krm Next time, let's avoid calling `this.safeNumber (interest, 'volume/'value)` multiple times, we can just memoize the value once and use it as many times as necessary. 

(I've fixed it now so nw)